extensions/amp-form/0.1/amp-form.js

```diff
@@ -1588,6 +1590,7 @@ export class AmpFormService {
       this.installSubmissionHandlers_(root.querySelectorAll('form'));
       AmpFormTextarea.install(ampdoc);
       this.installGlobalEventListener_(root);
```

Comment:
This function reinstalls the submission handlers on DOM changes. Should we add a hook in here too?

Reply:
Renamed to `installDomUpdateEventListener_`